tfhe/src/integer/gpu/server_key/radix/add.rs

```diff
@@ -208,4 +208,142 @@ impl CudaServerKey {
         }
         stream.synchronize();
     }
+
+    /// # Safety
+    ///
+    /// - `stream` __must__ be synchronized to guarantee computation has finished, and inputs must
+    ///   not be dropped until stream is synchronised
+    pub unsafe fn unchecked_sum_ciphertexts_assign_async(
```

Comment:
@tmontaigu regarding naming I don't know if this matches what you expect.

Reply:
looks ok